pkg/cluster/deploystorage_test.go

```diff
@@ -0,0 +1,86 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"reflect"
+	"testing"
+
+	mgmtauthorization "github.com/Azure/azure-sdk-for-go/services/preview/authorization/mgmt/2018-09-01-preview/authorization"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+)
+
+var daTestCases = []struct {
```

Comment:
nit: in future, please reduce scope - this can/should be defined in TestDenyAssignments